molecule/default/prepare.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Prepare
+  hosts: all
+  become: true
+  tasks:
+    - include_tasks: postgresql.yml
+    - include_tasks: apache.yml
+    - include_tasks: python.yml
```

Comment:
MiaRec web application depends on redis as well. If redis is not available the appliction will not function properly.
It is necessary to install redis as well.